### DISCUSSION TITLE
How to create a complicated form

### DISCUSSION THREAD

**Discussion by imakeitworknet** [statamic/cms#6746] [Q&A]
Hi. I need to build this kind of functionality in Statamic:

There are two collections: **Trips** and **Ships**.

Administrator needs to be able to add one or more ships in every single trip, and add a price. For example:

- trip 001 with ship A costs 100
- trip 001 with ship B costs 150

That's not an issue. Tricky thing is this: when client opens TRIP 001 page, he needs to fill a FORM (with name and other details), and needs to select one of the ships available in that trip.

System uses the same FORM in all Trips. The only difference is that the FORM needs to somehow also catch which ship was selected, which price it was showing at the moment, and the page title it was filled on (Trip name).

Any ideas how Form could catch those additional elements which are outside the form?

Thanks!

**Comment by imakeitworknet** [comment]
Never mind, sorted it with JS.

**Comment by jackmcdade** [comment]
For official support you can go to https://Statamic.com/support — this discussion area is community driven, so there’s no guarantee of anyone contributing an answer. Just wanted to give you a heads up!On Sep 20, 2022, at 11:17 PM, Hoa Anh SKS ***@***.***> wrote:﻿
I have the same question as you? Hope to get support
